trace/opencensus/trace/tracer/context_tracer.py

```diff
@@ -78,6 +78,21 @@ def set_enabled(self):
         else:
             return False
 
+    def trace_decorator(self):
+        """Decorator to trace a function."""
+
+        def decorator(func):
+
+            def wrapper(*args, **kwargs):
```

Comment:
Yes, that's correct. The calls to hello will add spans to the context belong to the decorator which is specified in `@tracer.trace_decorator()`. I think the decorator usage for tracing is intended to use in the client side to trace the function that makes a request to the server.